.github/workflows/analyzer.yaml

```diff
@@ -13,6 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+
+      - name: Load openvds image
+        uses: "./.github/actions/load_openvds_image"
+
       - name: Run check
         run: |
-          DOCKER_BUILDKIT=1 docker build -f Dockerfile --target static_analyzer .
+          DOCKER_BUILDKIT=1 docker build -f Dockerfile --build-arg OPENVDS_IMAGE=${{ env.OPENVDS_IMAGE_TAG }} --target static_analyzer .
```

Comment:
Question. Is it strictly needed to change the image name, or is it just a nicety ?

Reply:
Not sure what you mean by "change image name"? If you mean that in dockerfile it is named openvds and here it is named openvds:something, then I think it is needed as it must distinguish by name whether image is already there or if it needs to be built.